app/src/main/java/com/simply/birthdayapp/core/ZodiacName.kt

```diff
@@ -0,0 +1,16 @@
+package com.simply.birthdayapp.core
+
+object ZodiacName {
```

Comment:
you dont need this , you already create enum for this , you can keep your strings in resources and use id-s in enum